src/components/GoalCriteriaSuggest/GoalCriteriaSuggest.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState, useRef, useMemo, useCallback } from 'react';
+import { KeyCode, MenuItem, Popup, nullable, useKeyboard } from '@taskany/bricks';
+import { IconPlusCircleOutline } from '@taskany/icons';
+import styled from 'styled-components';
+
+import { trpc } from '../../utils/trpcClient';
+import { InlineTrigger } from '../InlineTrigger';
+import { CriteriaForm } from '../CriteriaFormV2/CriteriaForm';
+import { GoalBadge } from '../GoalBadge';
+
+import { tr } from './GoalCriteriaSuggest.i18n';
+
+type CriteriaFormValues = Parameters<React.ComponentProps<typeof CriteriaForm>['onSubmit']>[0];
+
+interface GoalCriteriaComboBoxProps {
+    onSubmit: (values: CriteriaFormValues) => void;
+}
+
+const StyledPopup = styled(Popup)`
+    z-index: 9999;
+    pointer-events: all;
```

Comment:
Maybe it would be better to add prop `interactive`? This should help

Reply:
If passing `interactive` prop, then popup will render as sibling of target element